content/guides/core-concepts/cypress-studio.md

```diff
@@ -2,17 +2,9 @@
 title: Cypress Studio
 ---
 
-<Alert type="warning">
+<E2EOnlyBadge class="my-10"></E2EOnlyBadge>
 
-<strong class="alert-header"><Icon name="exclamation-triangle"></Icon>
```

Comment:
It's probably worth keeping this banner up top to make it clear that this is still an experimental feature. We take the opportunity to do so on the [session API page](https://docs.cypress.io/api/commands/session). 

The verbiage used here could help simplify the first part of the where it's noted under the Using Cypress Studio section.